src/elm/Claim.elm

```diff
@@ -475,6 +500,172 @@ viewClaimCard loggedIn profileSummaries claim =
         ]
 
 
+type alias CompletionStatus =
+    { approved : Int
+    , disapproved : Int
+    , verifications : Int
+    }
+
+
+type VotingProgressBar
+    = ClaimCard
+    | ClaimModalAndDetailPage
+
+
+viewVotingProgress : Shared -> CompletionStatus -> VotingProgressBar -> Html msg
+viewVotingProgress shared completionStatus claimSpace =
+    let
+        { t } =
+            shared.translators
+
+        totalVotes =
+            toFloat completionStatus.verifications
+
+        approvedWidth =
+            (toFloat completionStatus.approved / totalVotes) * 100
+
+        disapprovedWidth =
+            (toFloat completionStatus.disapproved / totalVotes) * 100
+
+        votingLeft =
+            completionStatus.verifications - completionStatus.approved - completionStatus.disapproved
+
+        voteNumberTitleConditional compStatus =
+            if compStatus == 1 then
+                text (t "claim.vote")
+
+            else
+                text (t "claim.votes")
+
+        viewVotesBar =
+            div [ class "w-full h-2 bg-gray-500 flex rounded-full my-2" ]
+                [ div
+                    [ class "flex rounded-full overflow-hidden h-2"
+                    , style "width" (String.fromFloat (approvedWidth + disapprovedWidth) ++ "%")
+                    ]
+                    [ div [ class "bg-green", style "width" (String.fromFloat (toFloat completionStatus.approved / toFloat (completionStatus.approved + completionStatus.disapproved) * 100) ++ "%") ] []
+                    , div
+                        [ class "bg-red"
+                        , style "width" (String.fromFloat (toFloat completionStatus.disapproved / toFloat (completionStatus.approved + completionStatus.disapproved) * 100) ++ "%")
+                        ]
+                        []
+                    ]
+                ]
+    in
+    case claimSpace of
+        ClaimCard ->
+            div
+                [ class "flex flex-col mb-6" ]
+                [ div
+                    [ class "flex" ]
+                    [ if votingLeft == 0 then
+                        text ""
+
+                      else
+                        p
+                            [ class "w-full text-right text-gray-600" ]
+                            [ text (t "claim.pending") ]
+                    ]
+                , viewVotesBar
+                , div [ class "flex" ]
+                    [ if completionStatus.approved == 0 then
+                        text ""
+
+                      else
+                        p
+                            [ style "width" (String.fromFloat approvedWidth ++ "%") ]
+                            [ span [ class "font-bold text-green" ]
+                                [ text (String.fromInt completionStatus.approved ++ " ")
+                                ]
```

Comment:
Why the extra space? :thinking:

Reply:
Went through the cracks. Fixing right now.